sml_small/utils/pandas_example.py

```diff
@@ -84,9 +85,8 @@ def load_csv(filepath):
     upper_limit_column="threshold_upper",
     lower_limit_column="threshold_lower",
     predictive_column="predictive_val",
-    auxiliary_column="aux_val",
```

Comment:
Why aren't we passing auxiliary through?

Reply:
my bad I used this branch for the demo with Ellie last week and must have forgotten to undo the changes to this file when showing thousand pounds working with "true optional" values, amended